### PR TITLE
Add "Hall of Fame" emergency shuttle

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
This adds a new emergency shuttle called the **Hall of Fame**.

It's designed around persistence.  The goal is to have the shuttle store memories, photos, and trophies for the crew to see!

## Why It's Good For The Game
Cool way for the crew to store and share memories.

## Changelog
:cl:
add: Add a new 'Hall of Fame' emergency shuttle.  It even comes with it's own nifty photo album.
/:cl:

### CODE REVIEW

File: _maps/shuttles/emergency_fame.dmm [hunk 1]
```diff
@@ -0,0 +1,1908 @@
...
+"rH" = (
+/obj/structure/chair/sofa/corp/corner{
+	dir = 1;
+	name = "couch";
+	desc = "Looks like someone threw it out. Covered in donut crumbs."
+	},
+/turf/open/floor/mineral/silver,
+/area/shuttle/escape)
+"rV" = (
+/obj/structure/sign/picture_frame/showroom/one,
```
Comment (Mickyan) [inline _maps/shuttles/emergency_fame.dmm:423]: Currently these picture frames will mirror the persistence from the station showroom and will possibly end up overwriting each other when persistence is saved

What you may want to do instead is give each of them a unique persistence_id (i.e. fameship1, fameship2 etc.), no need to make them subtypes just varedits should be enough

**Comment by distributivgesetz** [comment]
add a tomb of the unknown employee, maybe?

**Comment by Addust** [comment]
> add a tomb of the unknown employee, maybe?

fun fact! this tomb is arclumin's tomb

**Comment by HoodMilk** [comment]
Nice looking shuttle that is very flavorful.

File: _maps/shuttles/emergency_fame.dmm (hunk 1 above)
Comment (timothymtorres) [reply _maps/shuttles/emergency_fame.dmm]: Thanks for pointing this out.  Fixed with https://github.com/tgstation/tgstation/pull/77074/commits/70fb87d672493485b1d32c22000713a17d1df06d

**Comment by DaxDupont** [comment]
> fun fact! this tomb is arclumin's tomb
> fun fact
Ah yes, a _fun fact_ about someone a lot of us knew and interacted with who died in a car crash

**Comment by timothymtorres** [comment]
Added the tomb of the unknown employee and tweaked the layout slightly.  This should be the final version, pending approval.

**Review by Maurukas — APPROVED** [review]

**Pull request merged**